packages/fault-detector/src/helpers.ts

```diff
@@ -1,11 +1,11 @@
 import { Contract, BigNumber } from 'ethers'
 import { Logger } from '@eth-optimism/common-ts'
 
-export interface OutputOracle<TSubmissionEventArgs> {
+export interface OutputOracle {
```

Comment:
not super important but technically this interface is no longer needed. My understanding of this interface was to abstract the differences`L2OutputOracle` and `StateCommitmentChain` into one.

Can we simply utilize the typescript ABI for this contract directly? It would provide better type safety as well

Reply:
fwiw this is not a blocking suggestion!